src/main.cpp

```diff
@@ -144,8 +144,7 @@ int main(int argc, char** argv)
         if (pwstdin) {
             // we always need consume a line of STDIN if --pw-stdin is set to clear out the
             // buffer for native messaging, even if the specified file does not exist
-            static QTextStream in(stdin, QIODevice::ReadOnly);
-            static QTextStream out(stdout, QIODevice::WriteOnly);
+	    QTextStream out(stdout, QIODevice::WriteOnly);
```

Comment:
Look at the comment, I believe we cannot remove the stdin line without breaking something (keepassxc-browser). Also your indentation is wrong.

Reply:
compare with https://github.com/keepassxreboot/keepassxc/commit/7000e80465eeda140c6e530f38ee2e33a904e2f8.
1.  when this moved to getPassword() the static requirement was dropped leading to the buffer loss issue.
1.  nothing else here reads from _in_; the mere fact of creating a QTextStream does not drain STDIN.  due to very narrow scope nothing here **can** read from _in_.  thankfully draining STDIN is now accomplished in getPassword().

Reply:
Ok just double checking. I thought the same things